Content.Shared/_RMC14/Xenonids/Parasite/InfectableComponent.cs

```diff
@@ -13,4 +13,11 @@ public sealed partial class InfectableComponent : Component
         [Sex.Male] = new SoundPathSpecifier("/Audio/_RMC14/Voice/Human/infected_male.ogg"),
         [Sex.Female] = new SoundPathSpecifier("/Audio/_RMC14/Voice/Human/infected_female.ogg")
     };
+
+    [DataField, AutoNetworkedField]
+    public Dictionary<Sex, SoundSpecifier> PreburstSound = new()
+    {
+        [Sex.Male] = new SoundCollectionSpecifier("RMCMalePreburstScreams", AudioParams.Default.WithVariation(0.05f)),
+        [Sex.Female] = new SoundCollectionSpecifier("RMCFemalePreburstScreams", AudioParams.Default.WithVariation(0.05f))
```

Comment:
Is the variation here intended? Whisper thinks it makes the screams sound strange

Reply:
Removed